### PR TITLE
IS_VALID_JSON typo fix

### PR DESCRIPTION
**Pull request by pmcneill**
Misplaced paren here, I believe.

### CODE REVIEW

**Comment by nodesocket** [comment]
Awwww, you are absolutely right. Would you mind bumping the `VERSION` file and also adding a note into `CHANGELOG.md` for the new version. Then I will merge the PR.

**Comment by pmcneill** [comment]
Quick response! :)  I've updated those files and amended my commit.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 CHANGELOG
 =========
 
+## V1.1.3 - *12/01/2015*
+
+- Fixes typo in assetBody(IS_VALID_JSON). Thanks [@pmcneill](https://github.com/pmcneill)
```
Comment (nodesocket) [inline CHANGELOG.md]: spelling error 'assert' :-)

**Comment by nodesocket** [comment]
Was this throwing a runtime syntax error, or just PHP swallowing this check?

**Comment by pmcneill** [comment]
The latter, more or less.  The "body === null" bit was getting evaluated first, probably always resulting in a false.  json_decode returns null when the input is invalid, so the if's condition will always be false.  The exception never gets thrown, so the result is it considers any input as valid JSON.

Fixed the spelling error -- thanks.

**Pull request merged**